src/exchanges/osmosisDex.ts

```diff
@@ -12,6 +12,10 @@ interface OsmosisOptions {
 
 const MAX_POOLS = 75
 
+function isGamm(object: OsmosisPool): object is OsmosisGammPool {
+  return 'pool_assets' in object
+}
+
```

Comment:
Let's place this logic in the `determineOsmosisPoolType` method... not sure why we would separate it. Additionally, is this a surefire way to determine whether it's a gamm pool? Seems that we check for pool_assets already

Reply:
Separated concentrated liquidity logic, so no requirement for it anymore